docs/src/main/sphinx/connector/hive-azure.rst

```diff
@@ -12,29 +12,51 @@ Trino supports both ADLS Gen1 and Gen2. With ADLS Gen2 now generally available,
 we recommend using ADLS Gen2. Learn more from `the official documentation
 <https://docs.microsoft.com/en-us/azure/data-lake-store/data-lake-store-overview>`_.
 
-Hive connector configuration
-----------------------------
+Hive connector configuration for Azure Storage credentials
+----------------------------------------------------------
+
+All of the configuration properties for the Azure storage driver are stored in
+the Hadoop ``core-site.xml`` configuration file. To configure Trino to use the
```

Comment:
Is that true.. does the connector actually create and update that file?

Reply:
@mfzarko can you confirm this is the functionality?

Reply:
@mosabua @rosewms 
Since there's no JIRA attached it's hard to follow what is the primary goal, I can only attest to technical scope here.

I can  confirm what Manfred said: `catalog.properties` take precedence before `core-site.xml`, which should be consumed only as last resort.
IIRC the plan was to retire `core-site.xml` entirely where possible, but just-in-case reference the values that **need** to be configured through it.
Rose, could you clarify the overall intention here?